### PR TITLE
feat(tree): API word

### PR DESCRIPTION
**Pull request by necoxrr**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

**Review by ErKeLost — APPROVED** [review]

File: packages/renderless/src/common/deps/tree-model/tree-store.ts [hunk 1]
```diff
@@ -88,7 +88,10 @@ export default class TreeStore {
     }
 
     const nodeKey = typeof data !== 'object' ? data : getNodeKey(this.key, data)
+<<<<<<< Updated upstream
```
Comment (gimmyhehe) [inline packages/renderless/src/common/deps/tree-model/tree-store.ts]: 冲突内容，需要解决一下冲突~

File: examples/sites/demos/pc/app/tree/webdoc/tree.js [hunk 2]
```diff
@@ -492,6 +492,57 @@ export default {
           `
       },
       codeFiles: ['other.vue']
+<<<<<<< Updated upstream
```
Comment (gimmyhehe) [inline examples/sites/demos/pc/app/tree/webdoc/tree.js]: 需要解决冲突

File: packages/renderless/src/tree/vue.ts [hunk 3]
```diff
@@ -226,7 +226,7 @@ const initApi = ({ state, dispatch, broadcast, props, vm, constants, t, emit, ap
   initIsCurrent: debounce(20, initIsCurrent({ props, state })),
   setCheckedByNodeKey: setCheckedByNodeKey({ props, state })
 })
-
+// 注释
```
Comment (gimmyhehe) [inline packages/renderless/src/tree/vue.ts]: 无效的注释，建议删除

File: packages/renderless/src/tree/index.ts [hunk 4]
```diff
@@ -1044,3 +1044,22 @@ export const setCheckedByNodeKey =
       plainNode.node.setChecked(checked, !checkStrictly)
     }
   }
+<<<<<<< Updated upstream
```
Comment (gimmyhehe) [inline packages/renderless/src/tree/index.ts]: 冲突内容，需要先解决冲突

File: packages/renderless/src/common/deps/tree-model/tree-store.ts (hunk 1 above)
Comment (necoxrr) [reply packages/renderless/src/common/deps/tree-model/tree-store.ts]: 已经解决

File: examples/sites/demos/pc/app/tree/webdoc/tree.js (hunk 2 above)
Comment (necoxrr) [reply examples/sites/demos/pc/app/tree/webdoc/tree.js]: 已经解决

File: packages/renderless/src/tree/vue.ts (hunk 3 above)
Comment (necoxrr) [reply packages/renderless/src/tree/vue.ts]: 好的

File: packages/renderless/src/tree/index.ts (hunk 4 above)
Comment (necoxrr) [reply packages/renderless/src/tree/index.ts]: 已经解决

**Comment by kagol** [comment]
@necoxrr 需要解决下冲突

**Review by kagol — APPROVED** [review]

**Pull request merged**